components/mbedtls_cxx/mbedtls_wrap.cpp

```diff
@@ -1,23 +1,36 @@
 /*
- * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ * SPDX-FileCopyrightText: 2023-2024 Espressif Systems (Shanghai) CO LTD
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#include <mbedtls/timing.h>
 #include "mbedtls/ctr_drbg.h"
 #include "mbedtls/ssl.h"
 #include "mbedtls_wrap.hpp"
 
-bool Tls::init(is_server server, do_verify verify)
+bool Tls::init(is_server server, do_verify verify, TlsConfig *config)
 {
     const char pers[] = "mbedtls_wrapper";
+    if (config == nullptr) {
+
```

Comment:
Missed return

Reply:
I'll remove the entire condition, as `config == nullptr` is a valid parameter